internal/flink/utils.go

```diff
@@ -0,0 +1,52 @@
+package flink
+
+import (
+	"bytes"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/dghubble/sling"
+)
+
+func uploadFile(url, filePath string, formFields map[string]any) error {
+	var buffer bytes.Buffer
+	writer := multipart.NewWriter(&buffer)
+
+	for key, value := range formFields {
+		if strValue, ok := value.(string); ok {
+			_ = writer.WriteField(key, strValue)
+		}
+	}
+
+	file, err := os.Open(filePath)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	part, err := writer.CreateFormFile("file", filepath.Base(filePath))
+	if err != nil {
+		return err
+	}
+	if _, err := io.Copy(part, file); err != nil {
+		return err
+	}
+
+	if err := writer.Close(); err != nil {
+		return err
+	}
+
+	client := &http.Client{
+		Timeout: 20 * time.Minute,
+	}
```

Comment:
Wow. How big are these files?

Reply:
```suggestion
	client := &http.Client{Timeout: 20 * time.Minute}
```

Reply:
normally it's <20MB in terms of file size. this time is cloned from https://github.com/confluentinc/cli/blob/3b0527b316868d961060f38b8a47e89f7ecd2a1e/internal/connect/utils.go#L118-L119